.github/workflows/sonarcloud.yaml

```diff
@@ -7,13 +7,10 @@ on:
   pull_request:
 
 jobs:
-  sonarcloud_query_user_services:
-    name: query-user-services
+  sonarcloud_home_backend:
+    name: home_backend
     runs-on: ubuntu-latest
     timeout-minutes: 15
-    defaults:
-      run:
-        working-directory: ./lambda/query-user-services
     steps:
```

Comment:
Some of these steps may not be required, around npm dependencies and coverage reporting. Docs here provide a code snippet with required steps https://govukverify.atlassian.net/wiki/spaces/PLAT/pages/3668280149/How+to+integrate+GitHub+repositories+with+SonarCloud

Which I've also done here https://github.com/govuk-one-login/account-management-stubs/blob/main/.github/workflows/sonarcloud-oidc-token.yaml

May be an improvement after initial refactor but one to note

Reply:
I have tidied up the github workflow for sonarqube.  If I remove the step that generates the coverage report, as per your referenced config, then the code coverage is flagged as 0%,